src/ja_romaji.qml

```diff
@@ -44,5 +77,10 @@ KeyboardLayout {
         BackspaceKey {}
     }
 
-    SpacebarRow { }
+    SpacebarRow {}
+
+    type: "custom"
```

Comment:
From what I can tell, isn't Component.onCompleted unneeded here? applyAutocaps in KeyboardBase.qml will be called immediately on startup, and as long as `type` isn't empty, autocaps will automatically be set to false.

Also, I think it might be good to set `type` to a somewhat more descriptive name, e.g. "japan_romaji_anthy" or something like that.